README.md

```diff
@@ -1 +1,56 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Branch should have a name of a feature or action e.g: ```created_sidebar``` or ```added_comments_section```, this will make your branches more organized

Reply:
hello @Ismailco, I just notice that if I changed my branch name it would close my pull request, what do you suggest to me please?

Reply:
Hi @faustM, Sorry for the inconvenience, you can ignore this in your changes for now but in the future please use a meaningful feature branch name

Reply:
thankyou